app/src/ui/toolbar/branch-dropdown.tsx

```diff
@@ -170,6 +173,20 @@ export class BranchDropdown extends React.Component<
     this.setState({ pullRequests })
   }
 
+  private get currentPullRequest(): IPullRequest | null {
```

Comment:
For the future it'd be real nice if we could get this into state and only recalculate when necessary

Reply:
Same as above, just placeholder.